support/collect/collect_test.go

```diff
@@ -0,0 +1,145 @@
+package collect
+
+import (
+	"github.com/stretchr/testify/assert"
+
+	"sort"
+	"strconv"
+	"testing"
```

Comment:
1. System packages
2. Third packages
3. Self packages

```suggestion
	"sort"
	"strconv"
	"testing"
	
	"github.com/stretchr/testify/assert"
```